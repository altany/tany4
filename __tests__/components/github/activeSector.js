import { render } from '@testing-library/react'
import ActiveSector from '../../../components/github/activeSector'


const props = {
    cx:296,
    cy:131.8046875,
    innerRadius:69.742578125,
    outerRadius:101.44375,
    startAngle:0,
    endAngle: 264.9082571453812,
    fill:'hsl(0,50%,65%)',
    payload: {name: 'Javascipt'},
    percent: 0.799721,
}

it('renders an active sector with language unchanged', () => {
  const { container } = render(<ActiveSector {...props} />)
  expect(container).toMatchSnapshot()
})

it('renders an active sector without language unchanged', () => {
    const { container } = render(<ActiveSector { ...props} payload={undefined} />)
    expect(container).toMatchSnapshot()
})