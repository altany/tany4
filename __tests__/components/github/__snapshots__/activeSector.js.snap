// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an active sector with language unchanged 1`] = `
<div>
  <svg>
    <g>
      <text
        dy="8"
        fill="hsl(0,50%,65%)"
        text-anchor="middle"
        x="296"
        y="131.8046875"
      >
        Javascipt 79.97%
      </text>
      <path
        class="recharts-sector"
        cx="296"
        cy="131.8046875"
        d="M 397.44375,131.8046875
    A 101.44375,101.44375,0,
    1,0,
    286.996790890694,232.8481268703847
  L 289.81029373735214,201.27205206713947
            A 69.742578125,69.742578125,0,
            1,1,
            365.742578125,131.8046875 Z"
        fill="hsl(0,50%,65%)"
      />
      <path
        class="recharts-sector"
        cx="296"
        cy="131.8046875"
        d="M 407.44375,131.8046875
    A 111.44375,111.44375,0,
    1,0,
    286.1092833695992,242.8086655305175
  L 286.4642863780371,238.82445006646438
            A 107.44375,107.44375,0,
            1,1,
            403.44375,131.8046875 Z"
        fill="hsl(0,50%,65%)"
      />
    </g>
  </svg>
</div>
`;

exports[`renders an active sector without language unchanged 1`] = `
<div>
  <svg>
    <g>
      <text
        dy="8"
        fill="hsl(0,50%,65%)"
        text-anchor="middle"
        x="296"
        y="131.8046875"
      >
        undefined 79.97%
      </text>
      <path
        class="recharts-sector"
        cx="296"
        cy="131.8046875"
        d="M 397.44375,131.8046875
    A 101.44375,101.44375,0,
    1,0,
    286.996790890694,232.8481268703847
  L 289.81029373735214,201.27205206713947
            A 69.742578125,69.742578125,0,
            1,1,
            365.742578125,131.8046875 Z"
        fill="hsl(0,50%,65%)"
      />
      <path
        class="recharts-sector"
        cx="296"
        cy="131.8046875"
        d="M 407.44375,131.8046875
    A 111.44375,111.44375,0,
    1,0,
    286.1092833695992,242.8086655305175
  L 286.4642863780371,238.82445006646438
            A 107.44375,107.44375,0,
            1,1,
            403.44375,131.8046875 Z"
        fill="hsl(0,50%,65%)"
      />
    </g>
  </svg>
</div>
`;
